script.js

```diff
@@ -1,48 +1,126 @@
 let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
-//ash says we can use this as a global object
+
 let gameState = ["", "", "", "", "", "", "", "", ""];
+let currentPlayer = "x";
+const tiles = document.querySelectorAll(".tile");
 
 function decideTurn(currentPlayer) {
-    if (currentPlayer === 'o') {
-        return 'x';
-    }
-    return 'o';
+  if (currentPlayer === "o") {
+    return "x";
+  }
+  return "o";
 }
+
 function addToScreen(tile, currentPlayer) {
-    tile.innerHTML = currentPlayer;
+  if (currentPlayer === "x") {
+    tile.classList.add("clickedX");
+  }
+  if (currentPlayer === "o") {
+    tile.classList.add("clickedO");
+  }
```

Comment:
Rather than a separate if statement, could this just be an else?

Reply:
done